basics/section_2-26.rb

```diff
@@ -0,0 +1,89 @@
+# Array
+
+a = [1, 2, 3, 4, 5, 6, 7, 8, 9]
+p a.last # 9
+
+# Using of ..
+# Create a Range of number inclusive of 100
+x = 1..100
+p x.class # Range
+p x.to_a # Show array consist of element 1 to 100
+
+# Shuffle the element in the array
+# The shuffle here is mutating the to_a instead of x
+p x.to_a.shuffle!
+p x # 1..100
+
+x = (1..10).to_a
+p x.reverse! # [10, 9, 8, 7, 6, 5, 4, 3, 2, 1]
+
+x = "a".."z"
+p x.to_a # ["a", "b", "c", "d", "e", "f", "g", "h", "i", "j", "k", "l", "m", "n", "o", "p", "q", "r", "s", "t", "u", "v", "w", "x", "y", "z"]
+
+# Shovel Operator
+# - add element to end of the Array
+a = [1, 2, 3, 4, 5, 6, 7, 8, 9]
+a << 10
+p a # [1, 2, 3, 4, 5, 6, 7, 8, 9, 10]
+
+# Add something to the first element in the Array
+p a.unshift("Name") # ["Name", 1, 2, 3, 4, 5, 6, 7, 8, 9, 10]
+
+# Add something to the last element in the Array
+p a.append("Name") # ["Name", 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, "Name"]
+
+# Remove duplicate element and make element unique
+# Does not mutate
+p a.uniq # ["Name", 1, 2, 3, 4, 5, 6, 7, 8, 9, 10]
+
+# Check if element is in the Aray
+p a.include?("Name") # true
+
+# Push the new element to the end of Array
+p a.push("New Item") # ["Name", 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, "Name", "New Item"]
+
+# Return the last element in the Array & remove it from array LILO (Last In Last Out)
+p b = a.pop
+
+# Join the element inside the Array
+# Does not mutate
+p a.join # "Name12345678910Name"
```

Comment:
This is `join` with what delimiter?

Reply:
Same as `.join("")` with `""` as delimiter.